src/config.ts

```diff
@@ -56,9 +58,7 @@ export const defaultRadius = 2000; //in meters
 export const allowLocalhostUI = Boolean(
   process.env.ALLOW_LOCALHOST_UI === 'true'
 );
-export const allowHardhatNetwork = Boolean(
-  process.env.ALLOW_HARDHAT_NETWORK === 'true'
-);
+export const networkMode = String(process.env.NETWORK_MODE) as NetworkMode;
 
 export const assetsCurrencies = ['EUR', 'USD', 'JPY', 'CNY']; // currency the asset is pegged to
```

Comment:
I recommend importing this definition from the `win-commons`, It is there, believe me